### PR TITLE
feat(python): add CTS

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-1710

### Changes included:

adds the requests assertions for the Python client.

I'll iterate on better naming/serializer in a subsequent PR because it's kind of a mess everywhere right now

### CODE REVIEW

File: templates/python/__init.client__.mustache
Comment (shortcuts) [inline templates/python/__init.client__.mustache:1]: sadly I had to drop barrel imports due to circular dependencies, I'll come back to it for DX improvements but that's not necessary for now

File: tests/output/python/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The python playground
```
Comment (millotp) [inline tests/output/python/README.md:1]:
```suggestion
The python Common Test Suite
```

**Review by millotp — COMMENTED** [review]
looks great, the CI is not pushing the tests however, maybe you need to edit `testsToStore` in `createMatrix.ts` ?

**Comment by millotp** [comment]
Screenshot 2023-12-18 at 18 15 53

ah yes you need to add python to the `createClientName`

**Review by millotp — APPROVED** [review]
gg let's go !

File: config/clients.config.json [hunk 2]
```diff
@@ -93,7 +93,7 @@
     "customGenerator": "algolia-ruby",
     "tests": {
       "extension": "_test.rb",
-      "outputFolder": "test"
+      "outputFolder": "tests"
```
Comment (millotp) [inline config/clients.config.json:96]: ah it was you that broke my CTS !

Comment (shortcuts) [reply config/clients.config.json:96]: Ahaha oopsie

**Pull request merged**